catalog/resource_catalog_test.go

```diff
@@ -38,7 +47,14 @@ func TestCatalogCreateAlsoDeletesDefaultSchema(t *testing.T) {
 						"c": "d",
 					},
 					MetastoreID: "e",
-					Owner:       "f",
+					Owner:       "administrators",
+				},
+			},
+			{
```

Comment:
can you also move this fixture a bit up, so it represents the real sequence of requests? sometimes i refrain to these tests as documentation for support, who probably could easier read API request flows, than go code.

the real sequence for create is
1. POST
2. DELETE
3. PATCH
4. GET

correct?